samples/subsys/usb/hid-mouse/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _usb_hid-mouse:
+
+USB HID mouse Sample Application
+####################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB HID driver provided by the zephyr
+project, enumerate a board into a mouse. But that mouse only has left button
+and can't move.
+This sample can be found under :file:`samples/subsys/usb/hid-mouse` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver, and there must has at least one
+gpio button in your board.
+
+Running
+********************
+
+Plug the board into a host device, for example, a PC running Linux.
+The board will be detected as shown by the Linux dmesg command:
+
+.. code-block:: console
+
+    usb 2-2: new full-speed USB device number 2 using at91_ohci
```

Comment:
Let's put the command in too (for example):

    dmesg -tail 10
    usb 2-2: new full-speed ....